test/atomics.wast

```diff
@@ -134,24 +134,24 @@
    )
   )
  )
- (func $atomic-wait-wake (type $0)
+ (func $atomic-wait-atomic.notify (type $0)
```

Comment:
This name seems a little strange now. Maybe `atomic-wait-notify`?

Reply:
Done. Auto search-replace was the culprit :$